test/built-ins/RegExp/prototype/Symbol.match/get-sticky-err.js

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: Error thrown when accessing `sticky` property
```

Comment:
This makes it seem like the exception is **always** thrown. We should make it clear in the description that it is thrown when the accessor throws it.